File: Nim/factors.nim
import strformat
from times import nil

let start = times.cpuTime()
const num: int = 2000000000

for i in 1..num:
    if num mod i == 0:
        echo(fmt"{i} is a factor of {num}")

echo(fmt"Time taken: {times.cpuTime()-start}s")

File: Lua/factors.lua
start = os.clock()

num = 2000000000

for i = 1, num+1 do
	if num % i == 0 then
		print(i.. " is a factor of "..num)
	end
end

print(string.format("Time taken: %.2fs\n", os.clock() - start))
